ibmcloud/README.md

```diff
@@ -314,18 +405,20 @@ After `terraform apply` completes the `cloud-api-adaptor` process will run on th
 
 ### Deploy the nginx pod and sniff test nginx
 
-A Terraform template that will deploy an nginx pod to the Kubernetes cluster is available in [ibmcloud/terraform/run-nginx-demo](./terraform/run-nginx-demo).
+A Terraform configuration that will deploy an nginx pod to the Kubernetes cluster is available in [ibmcloud/terraform/run-nginx-demo](./terraform/run-nginx-demo). This configuration will also check the nginx peer pod virtual server instance has been successfully created.
 
-Create a `terraform.tfvars` file in the [template directory](./terraform/run-nginx-demo) for this Terraform template on your `development machine`. The `terraform.tfvars` file should look like this
+Create a `terraform.tfvars` file in the [configuration directory](./terraform/run-nginx-demo) for this Terraform configuration on your `development machine`. The `terraform.tfvars` file should look like this
 
 ```
 ibmcloud_api_key = "<your API Key>"
 cluster_name = "<cluster name>"
+vpc_name = "<name of your VPC>" OR vpc_id = "<ID of your VPC>"
 ```
 
 > **Notes:**
-> - `ibmcloud_api_key` is your IBM Cloud API Key that you created at [https://cloud.ibm.com/iam/apikeys](https://cloud.ibm.com/iam/apikeys).
-> - `cluster_name` is a name of a Kubernetes cluster. You must use the same value for this parameter as you used for the corresponding parameter when running the Terraform template in [ibmcloud/terraform/cluster](terraform/cluster) to create the cluster.
+> - The `vpc` resource can be specified using its name or ID. The `vpc_name` and `vpc_id` variables are mutually exclusive.
```

Comment:
Optional/idea: Similar to the above comments about whether we want to point the user to the VPC output variables?

Reply:
I did the same thing here as I did in the https://github.com/confidential-containers/cloud-api-adaptor/pull/49#discussion_r893355021 comment.